client/classic/upe/payment-processing.js

```diff
@@ -265,7 +265,17 @@ export const processPayment = (
 		} catch ( err ) {
 			hasCheckoutCompleted = false;
 			jQueryForm.removeClass( 'processing' ).unblock();
-			showErrorCheckout( err.message );
+
+			let errorMessage = err.message;
+			if ( err.code === 'parameter_invalid_empty' ) {
+				const param = err.param.match( /country|postalCode/ );
+				if ( param ) {
+					errorMessage = `Missing required billing address field: ${ param[ 0 ] }.`;
```

Comment:
We should use `__` and `sprintf()` from `@wordpress/i18n` to make sure these messages are translatable.

Reply:
Good catch, thanks! Added i18n in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3588/commits/afda7597077a8e3a803f1d6f069fbcf3f2729138.